util/fileglob.go

```diff
@@ -8,12 +8,26 @@ import (
 	"os/exec"
 	"path"
 	"strings"
+
+	"github.com/karrick/godirwalk"
 )
 
+// ListFilesGlob is the legacy, slower version that uses
+// the node glob tool found here: https://github.com/cdnjs/glob
 func ListFilesGlob(ctx context.Context, base string, pattern string) []string {
+	list := make([]string, 0)
+
+	// Currently not ignoring hidden files, uncomment out to compare with ListFilesInVersion directly
+
+	// // check if the version is hidden
+	// if isHidden(base) {
```

Comment:
I think dotfiles are safe to ignore. Here's the list https://gist.github.com/xtuc/299faacf84ea9b885c5dbedce7ad362b

Reply:
updated legacy ListFilesGlob function to ignore them